integration/util/scenario/starport.js

```diff
@@ -165,33 +169,35 @@ class Starport {
   }
 }
 
-async function buildStarport(starportInfo, validatorsInfoHash, ctx) {
-  ctx.log("Deploying Starport...");
-  if (!ctx.cashToken) {
-    throw new Error(`Cannot deploy Starport without first deploying Cash Token`);
-  }
+async function buildStarport(starportInfo, validatorsInfoHash, ctx, chain, cashToken) {
+  ctx.log(`Deploying Starport to ${chain.name}...`);
+
+  // xxx todo:wn add back sanity check (?)
+  // if (!ctx.cashToken) {
```

Comment:
I assume this check was there since we _needed it_. If it's not required, we can probably remove it.